doc/users/faq/howto_faq.rst

```diff
@@ -9,6 +9,61 @@ How-to
 .. contents::
    :backlinks: none
 
+
+.. _how-to-too-many-ticks:
+
+Why do I have so many ticks, and/or why are they out of order?
+--------------------------------------------------------------
+
+One common cause for unexpected tick behavior is passing a list of strings
+instead of numbers or datetime objects. This can easily happen without notice
+when reading in a comma-delimited text file. Matplotlib treats lists of strings
+as "categorical" variables
```

Comment:
I think italics is is more suited than quotes. Italics are often used for technical terms or emphasis, both of which intentions work here. Quotes are used for [many things](https://thevisualcommunicationguy.com/2013/09/11/10-things-you-really-need-to-know-about-quotation-marks/) but IMHO nothing applies here.

```suggestion
as *categorical* variables
```